content/blog/gatbsy-typescript/index.md

```diff
@@ -0,0 +1,284 @@
+---
+title: Adding Typescript to your Gatbsy application
+date: "2020-05-10"
+author: franleplant
+description: "Docker has become a widely used
+  technology and chances are you are going to have to deal with it eventually, at least superficially, in your Front End
+  career. Let's cover the basic concepts and day to day useful commands you will likely use when dealing with Docker."
+tags:
+  - gatsbyjs
+  - javascript
+  - react
+  - typescript
+  - graphql
+---
+
+## Intro
+
+I have been using Typescript for the past 4 years in different production
+projects in different companies and although this post is not meant to be
+a _why typescript?_ post let me tell you briefly why I love typescript
+and why I do think that most medium to largse size Javascript applications
+would benefit from using Typescript, in no particular order:
+
+1- automatically up-to-date and synchronized code documentation in the form of types.
+2- new code verification tool added to your pool (in javascript apps this is mostly made of different kinds of tests and lints)
+3- better development experience: better autocompletion and api discovery
+4- type checker guided hassle free refactoring
+5- improve the mind set of developers so that they think more about data structures and their semantic meaning inside the code (as oposed to just have a bunch of unamed and uncategorized objects which is something that tends to happen in bigger javascript apps).
```

Comment:
```suggestion
5- improve the mindset of developers so that they think more about data structures and their semantic meaning inside the code (as opposed to just have a bunch of unnamed and uncategorized objects which is something that tends to happen in bigger javascript apps).
```

Reply:
I think you start making a "review" when I first submit the PR and then finish it when I tell you it is ready to be reviewed (which might take couple of days). This makes your comments be on an outdated piece of code. 

You might want to make these comments as "simple comments" instead of reviews or simply submit reviews early on so that your comments don't get outdated.

Or... we might not be using github's UI correctly jajaja